code/game/objects/effects/spawners/lootdrop.dm

```diff
@@ -524,3 +524,12 @@
 		/obj/item/stack/ore/titanium = 2,
 		/obj/item/stack/ore/uranium = 2,
 		/obj/item/stack/ore/diamond = 2)
+
+
+/obj/effect/spawner/lootdrop/Engines
```

Comment:
I'd prefer if these effects were in the modular Monkestation folder.
Create if needed:
monkestation/code/game/objects/effects/spawners/lootdrop.dm